pkgs/tools/misc/grub/pvgrub_image/configs/grub.cfg

```diff
@@ -0,0 +1,9 @@
+if search -s -f /boot/grub/grub.cfg ; then
+	echo "Reading (${root})/boot/grub/grub.cfg"
```

Comment:
Regarding the parentheses, adding the comment 
`The parentheses around ${root} here to match Grub's config file syntax` to the source code would work wonders.